app/frontend/components/topNavigation/topNavigation.tsx

```diff
@@ -1,8 +1,12 @@
 import { signIn, signOut, useSession } from 'next-auth/react'
 import { TopNavigationLink } from './topNavigationLink'
 
+const handleOnUnauthenticated = () => {
+  console.log('handleOnUnauthenticated')
+}
+
 export const TopNavigation = (): JSX.Element => {
-  const session = useSession()
+  const session = useSession({ required: false, onUnauthenticated: handleOnUnauthenticated })
```

Comment:
why?

Reply:
removed, had version problems and some newer version required the options.